pipelines/fbc-builder/patch.yaml

```diff
@@ -26,3 +26,38 @@
     value: "$(tasks.appstudio-configure-build.results.buildah-auth-param)"
   - name: PUSH_EXTRA_ARGS
     value: "$(tasks.appstudio-configure-build.results.buildah-auth-param)"
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: fbc-validate
+    when:
+    - input: $(params.skip-checks)
+      operator: in
+      values: ["false"]
+    runAfter:
+      - sanity-label-check
```

Comment:
why after sanity-label-check?

Reply:
We need to make sure that image consists of label: `operators.operatorframework.io.index.configs.v1`
So there are configs to run tests on.